example/actions_link.dart

```diff
@@ -27,7 +27,9 @@ main(List<String> args) async {
   link.connect();
 
   // Save the message when it changes.
-  link.onValueChange("/message").listen((update) => link.save());
+  if (link.persistValue) {
```

Comment:
code looks all right, although, I'm sure sure I like the name. `persistValue` doesn't really suggest a state query, but rather an action.